README.md

```diff
@@ -1,66 +1,201 @@
-# binge - version-less JavaScript module management for monorepos
+# binge - version-less JavaScript package management for monorepos
+Binge is a version-less JavaScript package management tool for monolithic repositories (*monorepos*).
 
-the problem: managing modules with consistent dependencies in monorepo
-manage build scripts with as little duplication as possible
-avoid publishing/versioning overhead (Lerna!?)
+It helps you to:
+* encourage modulization for a clearer separation of concerns,
+* enforces dependency version consistency across different local packages,
+* improves the developer experience and reduces the build file size by enforcing there is only one version of a dependency across all packages in the repository.
 
```

Comment:
Concerning the intro, several points:
- Lets remove the monolithic word, it brings a bad cling with it. Maybe just monorepos.
- Then Important to touch on the following topics: (1) Modularization (2) Harmony (3) kill dependency hell instead of size (4) put some focus on the connecting nature of binge (you write your own builds and watches binge just puts it together).

So I am not sure about it, maybe you can massage it a bit, but a candidate for this section could be:

Binge is a version-less JavaScript package management tool for *monorepos*. 

It encourages:
- Separation of concerns: create and consume a local package in less than 60 seconds
- Dependency consistency: eliminates the dependency hell problem by enforcing external package harmony
- Improved developer experience: binge delivers local-package content into node_modules seamlessly and respecting npm semantics 

It helps you to:
+*  Overcome the publishing overhead: you get the same behaviour as a standalone npm module, without the publish overhead
+* Keep custom lifecycle steps (testing, building, linting) without the publishing overhead. Each package declares (optional) build and watch npm scripts, binge connects everything into the consumer's node_modules using only those interaction points.